### PR TITLE
docs: Add Careem to the user's list

### PR DESCRIPTION
**Pull request by SamraHanifCareem**
Adding 'Samra Hanif'/Careem to the user's list

### CODE REVIEW

**Review by villebro — CHANGES_REQUESTED** [review]
Thanks for adding Careem to the list! 🎉  LGTM, but please add the entry in the correct place (the list is alphabetically sorted)

**Comment by SamraHanifCareem** [comment]
> 

Incorporated

File: RESOURCES/INTHEWILD.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ Join our growing community!
 - [Avesta Technologies](https://avestatechnologies.com/) [@TheRum]
 - [Cloudsmith](https://cloudsmith.io) [@alancarson]
 - [CnOvit](http://www.cnovit.com/) [@xieshaohu]
+- [Careem](https://www.careem.com/) [@SamraHanifCareem]
```
Comment (villebro) [inline RESOURCES/INTHEWILD.md:77]: Two more notches up please (before Cloudsmith) 🙂

Comment (SamraHanifCareem) [reply RESOURCES/INTHEWILD.md]: Incorporated. Thanks @villebro

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**